modules/fundamental/src/vulnerability/service/test.rs

```diff
@@ -207,11 +207,35 @@ async fn commons_compress(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
 #[test_context(TrustifyContext)]
 #[test(actix_web::test)]
 async fn product_statuses(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
-    let service = VulnerabilityService::new(ctx.db.clone());
+    let vuln_service = VulnerabilityService::new(ctx.db.clone());
+    let sbom_service = SbomService::new(ctx.db.clone());
 
-    ctx.ingest_documents(["csaf/cve-2023-0044.json"]).await?;
+    let ingest_results = ctx
+        .ingest_documents([
+            "csaf/cve-2023-0044.json",
+            "quarkus/v2/quarkus-bom-2.13.8.Final-redhat-00004.json",
```

Comment:
Considering how much variation there is with sboms ... might be worthwhile to test a few different ones (quarkus is pretty opinionated).

Reply:
Yes, that's where #986 should come to play

Reply:
If it's at all possible to test the variations in a unit test, i'd prefer that to #986

Reply:
Let's definitely add more tests here as we go along and find the cases that we want to cover while improving the matching logic